js/parts-more/Polar.js

```diff
@@ -397,77 +413,53 @@ if (seriesTypes.column) {
                                 if (start < 0) {
                                     start = 0;
                                 }
+                                else if (start > visibleRange) {
+                                    start = visibleRange;
+                                }
                             }
                         }
                     }
                     else {
-                        // Calculating starting and ending angles for
-                        // inverted polar bars
-                        start = 0;
+                        // Initial start and end angles for radial bar
+                        start = thresholdAngleRad;
                         end = point.plotY;
                     }
-                    // Prevent from rendering point outside acceptable
-                    // circle range
                     if (start > end) {
-                        yMin = yAxis.translate(yAxis.min);
-                        yMax = yAxis.translate(yAxis.max);
-                        if (!yAxis.reversed) {
-                            if (start < yMin) {
-                                start = end = 0;
-                            }
-                            else if (end < yMin) {
-                                end = yMin;
-                            }
+                        _a = __read([end, start], 2), start = _a[0], end = _a[1];
```

Comment:
Please add comment what it is

Reply:
I have used here destructuring arrays to swap variables. I have added a fitting comment.

Reply:
Unfortunately this polyfill is inserted to support destructuring arrays. The problem with this is that so much junk code is inserted in order to simplify the simple task of swapping variables. Also, the current bundler isn't smart enough to add this to the upper scope, so it will potentially be added to every module. Long story short, we'll have to wait a while with array destructuring, and for the time being swap variables the old way :(

Reply:
Corrected, thanks!